deployments/gateway-istio.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: captivate-gateway
+spec:
+  selector:
+    istio: ingressgateway 
+  servers:
+  - port:
+      number: 80
+      name: http
+      protocol: HTTP
+    hosts:
+    - "*"
+
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: api-gateway
+spec:
+  hosts:
+  - api-gateway
+  gateways:
+  - captivate-gateway
+  http:
+  - route:
+    - destination:
+       host: api-gateway
```

Comment:
what is the purpose of this ?

Reply:
can be removed if we don't want to define traffic rules for this service